lib/collection/src/collection_manager/holders/proxy_segment.rs

```diff
@@ -535,6 +535,21 @@ impl SegmentEntry for ProxySegment {
         (wrapped_segment_count + write_segment_count).saturating_sub(deleted_points_count)
     }
 
+    fn available_vector_count(&self, vector_name: &str) -> OperationResult<usize> {
+        let deleted_points_count = self.deleted_points.read().len();
+        let wrapped_segment_count = self
+            .wrapped_segment
+            .get()
+            .read()
+            .available_vector_count(vector_name)?;
+        let write_segment_count = self
+            .write_segment
+            .get()
+            .read()
+            .available_vector_count(vector_name)?;
+        Ok((wrapped_segment_count + write_segment_count).saturating_sub(deleted_points_count))
+    }
```

Comment:
Not 100% sure about this one. Quickly sourced this from `available_point_count()`.